float64 er
float64 pr
float64[] er_k
float64[] pr_k
float64[] pk_joule
float64[] pk_mech
float64[] pk_indct
float64[] ek_joule
float64[] ek_mech
float64[] ek_indct

string[] iq_jnt_names

float64[] recov_energy
float64 recov_energy_tot
bool monitor_recov_energy

float64[] q_model
float64[] stiffness_setpoint
float64[] damping_setpoint
float64[] q_ref
float64[] q_dot_ref
float64 imp_severity_ratio

